README.md

```diff
@@ -683,7 +683,7 @@ It's necessary that Etcd be listening on an interface that can be accessed by ot
 First configure backend Etcd to listen on an interface besides localhost and also use SSL:
 
 ```puppet
-class { '::sensu::backend':
+class { 'sensu::backend':
```

Comment:
I believe the arrows are supposed to be aligned within the hash. Suggest copying these blocks to files and running puppet-lint on them.

Reply:
I think hashed parameters don't have to align but does look nicer so fixed. Several of my modules have hash variables that don't align and linting doesn't complain. It will if the hash arrows in class parameters don't align when doing `class { '...': }` but if the parameter contains a hash those arrows don't have to align.